api-gateway/src/main/resources/static/scripts/owner-details/owner-details.controller.js

```diff
@@ -7,4 +7,5 @@ angular.module('ownerDetails')
         $http.get('api/gateway/owners/' + $stateParams.ownerId).then(function (resp) {
             self.owner = resp.data;
         });
+
```

Comment:
[nitpick] in the future try to remove file changes of just a newline from the pr unless it is required